ibm/mas_devops/playbooks/cp4d/install-services-fullstack.yml

```diff
@@ -2,8 +2,14 @@
 - hosts: localhost
   any_errors_fatal: true
   vars:
+    # Cluster details
+    # ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------
+    cluster_name: "{{ lookup('env', 'CLUSTER_NAME') }}"
```

Comment:
Why do we need the CLUSTER_NAME and CLUSTER_TYPE for installing cpd services?

Reply:
Fixed. Removed CLUSTER_NAME, not really needed. Maintained CLUSTER_TYPE due ROKS limitation check i.e reload nodes.